cmd/goplugin-heroku/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"os"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/lyraproj/issue/issue"
+	"github.com/lyraproj/lyra/cmd/goplugin-heroku/heroku"
+)
+
+func init() {
+	// Tell issue reporting to amend all errors with a stack trace
+	if hclog.DefaultOptions.Level <= hclog.Debug {
```

Comment:
This if-statement must be placed after the setting of `hclog.DefaultOptions` or it will be insensitive to the setting of "LYRA__LOG_LEVEL"